pages/linux/genfstab.md

```diff
@@ -0,0 +1,20 @@
+# genfstab
+
+> Generate output suitable for addition to an fstab file.
+> More information: <https://man.archlinux.org/man/extra/arch-install-scripts/genfstab.8>.
+
+- Generate the output based on LABEL:
+
+`genfstab -L {{path/to/mount_point}}`
+
+- Display `fstab` output using partitions' UUID:
+
+`genfstab -U {{path/to/mount_point}}`
+
+- Generate a fstab file, requires being a superuser:
+
+`genfstab -U {{/mnt}} >> {{/mnt/etc/fstab}}`
```

Comment:
See my suggestion above.
```suggestion
`sudo genfstab -U {{/mnt}} >> {{/mnt/etc/fstab}}`
```

Reply:
This suggestion will fail, for sudo there is the line just below.
Thiis line in specifically used when superuser and it is the 99% of it's use cases, I would definitely not edit it in this way.

Reply:
By "as superuser", do you mean after having run something like `su`?

Reply:
If it's required to run the command as superuser as you say, why not prefix it with `sudo`?  We do that in many other pages, particularly in low-level file-system utilities, such as [`partx.md`](https://github.com/tldr-pages/tldr/blob/master/pages/linux/partx.md).

Reply:
To your First question, yes, for instance `su` is necessary to elevate privileges.

To your second question, the command will fail because of the redirection operator `>>`. Prefixing the whole line with with `sudo` will not work. To use redirection operator with elevated privileges, a different approach is necessary.

https://linuxize.com/post/bash-append-to-file/#append-to-a-file-using-the-redirection-operator-
https://www.cyberciti.biz/faq/sudo-append-data-text-to-file-on-linux-unix-macos/

Reply:
Ah, I see!  Makes perfect sense now.  Please disregard this suggestion.

Reply:
Hrm, I see the problem here. I would be inclined to go with @peterbabic's suggestion here, with something like this:

```suggestion
`genfstab -U {{path/to/mount_point}} | sudo tee -a /etc/fstab`
```

The reason for this is that by convention in other pages, we use `sudo` to indicate that a given command needs to be run as root - so I'd prefer to use that same convention here too if possible.

Also, welcome to GitHub and tldr-pages, @peterbabic!

Reply:
@sbrl this has already been resolved on another long thread on this same line :) - https://github.com/tldr-pages/tldr/pull/5215#discussion_r567993244